lib/node/fullnode.js

```diff
@@ -41,10 +42,20 @@ class FullNode extends Node {
     // SPV flag.
     this.spv = false;
 
+    // Instantiate block storage.
+    this.blocks = blockstore.create({
+      network: this.network,
+      logger: this.logger,
+      prefix: this.config.prefix,
+      cacheSize: this.config.mb('block-cache-size'),
+      memory: this.memory
```

Comment:
I was confused about this at first until I realized `this.memory` is inherited from the `node` parent class. I don't think this needs to be addressed in this PR since we are so close to merging, but we could probably replace all the memory settings with `this.memory` in the other modules like `chain` and `pool`.